terraform/runner/main.tf

```diff
@@ -8,10 +8,11 @@ module "object_store_instance" {
   cf_org_name      = var.cf_org_name
   cf_space_name    = var.cf_space_name
   name             = var.object_store_instance
+  s3_plan_name     = "basic-sandbox"
 }
 
 resource "cloudfoundry_app" "gitlab-runner" {
-    name = var.runner_name
+    name = var.app_name
```

Comment:
[question (nonblocking)] What do you think about going back to `runner_name` as the variable name? My default for thinking about `app_name` is the name of the application that the runner will eventually be deploying, so I think this might end up being confusing if app teams are ever interacting with this terraform

Reply:
I am fine with it. This follows the current manifest.yml, which has it hard coded to "gitlab-runner". If we keep it as runner name then I would also change change the manifest.yml to pull runner name from var.yml. @pauldoomgov any objections to having the runner name == app name?

Reply:
I like "runner_name" as the key name. Helps prevent confusion.

Reply:
so, in the original code runner name is set to glrx-blablabla and app name is gitlab-runner. We can't deploy multiple runners in cloudfoundry with the app name gitlab runner.

If I set this resource to runner name then we won't be able to name the cloudfoundry_app. It will be glrx-blablabla instead of whatever convention we decide for the fleet of runners. Are the long names intended for the cloud foundry app as well?